source/_integrations/zwave_js.markdown

```diff
@@ -44,18 +44,48 @@ ha_integration_type: hub
 ha_zeroconf: true
 ---
 
-This integration allows you to control a Z-Wave network via the [Z-Wave JS](https://zwave-js.github.io/node-zwave-js/#/) driver.
+The Z-Wave integration allows you to control a Z-Wave network via the [Node Z-Wave JS](https://zwave-js.github.io/node-zwave-js/#/) driver.
```

Comment:
```suggestion
The Z-Wave integration allows you to control a Z-Wave network via the [Z-Wave JS](https://zwave-js.github.io/node-zwave-js/#/) driver.
```
There's no "Node" in the official name, just in the URL (because it is a Node.js project)